providers/provider-pojo/src/main/java/org/apache/servicecomb/provider/pojo/Invoker.java

```diff
@@ -158,6 +161,16 @@ public Object invoke(Object proxy, Method method, Object[] args) {
     return syncInvoke(invocation, consumerOperation);
   }
 
+  protected String findSwaggerMethodName(Method consumerMethod) {
+    ApiOperation apiOperationAnnotation = consumerMethod.getAnnotation(ApiOperation.class);
+    if (apiOperationAnnotation == null || org.apache.commons.lang3.StringUtils
+        .isEmpty(apiOperationAnnotation.nickname())) {
```

Comment:
Hi, since the `org.springframework.util.StringUtils` has been imported, there is no need to use `org.apache.commons.lang3.StringUtils`

Reply:
ok